packages/react-magma-dom/src/theme/magma.ts

```diff
@@ -563,6 +563,25 @@ export const magma = {
     },
   },
 
+  charts: {
+    line: {
+      colors: [
```

Comment:
I'm guessing these don't match up with `magma.colors`?

Reply:
are the colors for `line` charts different than `bar` or `pie` or any other chart? 

Is it more important that these are colors, or for charts (or even for line charts directly)? 

If it's more important that they're a set of colors that work with the theme... but aren't the theme colors... maybe it makes more sense to just have an `iterableColors` array in the magma theme... then any component that just needs a set of colors that work with the theme can use it...